html/assets/js/tools/theme.js

```diff
@@ -2,21 +2,28 @@
 const setTheme = (theme) => {
     document.documentElement.className = theme;
     localStorage.setItem('theme', theme);
+
+    if (typeof setDisplaySettings === "function")
+      setDisplaySettings(theme);
 }
 
-// On load -> Prepare the Color Theme
+// On load -> Set the Color Theme
+window.matchMedia("(prefers-color-scheme: dark)").matches ? setTheme("dark") : setTheme("light");
+
+// On load -> listen for prefers-color-scheme change
 window.matchMedia("(prefers-color-scheme: dark)").addEventListener(
-    "change", 
-     e => e.matches && setTheme("dark")
-);      
+    "change",
+    e => setTheme(e.matches ? "dark" : "light")
+);
 
 // On load -> Check if there is a theme stored already
-var theme = localStorage.getItem('theme');
+let theme = localStorage.getItem('theme');
 theme && setTheme(theme);
 
 // Updates theme when changed by another tab (or console)
-const themeUpdater = window.setInterval(() => {
-    theme = localStorage.getItem('theme');
-    if (theme !== null)
-        setTheme(theme);
-}, 500);
\ No newline at end of file
+window.addEventListener("storage", ()=>{
+  const newTheme = localStorage.getItem("theme");
+
+  if (!!newTheme)
```

Comment:
Typo? Or is this some js feature, semantically different from `if (new theme)`

Reply:
The double not operator forces conversion of the value to it's boolean equivalent instead of being truthy or falsy.

If you think it makes the code harder to read, I can revert that. It's more of an habit of mine I took when coding with the AirBnB style guide than a necessity here.

Reply:
After properly checking with the style guide, it's unnecessary there. Will remove.